### DISCUSSION TITLE
目前label支持自定义ReactNode?

### DISCUSSION THREAD

**Discussion by ahuigo** [antvis/X6#3567] [Q&A] [answered]
查了下文档，看起来只支持字符串label：
https://x6.antv.antgroup.com/api/model/labels#%E5%AD%97%E7%AC%A6%E4%B8%B2%E6%A0%87%E7%AD%BE

不支持label react node.

**Comment by ahuigo** [comment, marked as answer]
示例显示支持了：
https://x6.antv.antgroup.com/examples/edge/label/#render-with-react

**Reply by komagic** [reply]
edge reactnode的更新太恶心了
